src/style.css

```diff
@@ -0,0 +1,66 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
```

Comment:
do you really need to reset these styles in all elements?

Reply:
tests fail without  reset :(

Reply:
do you really need to reset these styles in **ALL** elements?